### DISCUSSION TITLE
Possible to remove releases from the front page?

### DISCUSSION THREAD

**Discussion by Drakota** [community/community#4143] [Other Feature Feedback, Questions, & Ideas]
At the moment, by default my front page gets polluted by releases from repo I've starred. I've not specified I wanted those repos to have their releases notified to me.

Example:
![image](https://user-images.githubusercontent.com/22665836/121440297-a013fe00-c955-11eb-97ff-96ee0a5af28d.png)

**Comment by twe4ked** [comment]
Agreed. I've starred over 1500 repos, this is very noisy for me!

**Comment by twe4ked** [comment]
Any chance we can get an official response to this? I've had to start un-starring repos left, right, and center. I really feel like we've already got a great feature to be notified of releases (watching). I'm sure many others use GitHub stars as a way to bookmark projects and not as a way to subscribe to all updates about them.
